recipes/cellsnake/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "cellsnake" %}
+{% set version = "0.2.0.dev7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cellsnake-{{ version }}.tar.gz
+  sha256: a39201cec1df95fdccc00b13fe1e59026d2147f1d0ade5c07f96a853490d07d0
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - cellsnake=cellsnake.command_line:main
+
+channels:
+  - conda-forge
+  - anaconda
+  - bioconda
+
+
+requirements:
+  host:
+    - pip
+    - python ==3.10.9
```

Comment:
Are you sure you want to pin to this very specific version? It will limit what else can be installed in the environment with this tool, limiting usability.